azure-pipelines-library.yml

```
@@ -1,98 +1,21 @@
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      endpoint: piomin
+      name: piomin/azure-pipelines-templates
+      ref: refs/heads/master
+
 trigger:
 - master
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 0

(indentation)